src/Components/CommentsCard.jsx

```diff
@@ -1,10 +1,33 @@
-import React from 'react';
+import React, {useState, useContext} from 'react';
+import { deleteComment } from '../../api';
+import UserContext from '../Contexts/UserContext';
 
-const CommentsCard = ({comment}) => {
+const CommentsCard = ({comment, setComments, comments}) => {
 
+    const { currentUser } = useContext(UserContext)
+
+    const [isLoading, setIsLoading] = useState(false)
+
+
+    const deleteHandler = (event) => {
+    event.preventDefault()
```

Comment:
Is this needed? This button is not part of a form so it's default behaviour will have no effect